cypress/e2e/contactForm.cy.js

```diff
@@ -1,16 +1,16 @@
 import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 import HomeAndCataloguePageObject
-  from '../support/pages/homeСatalogue.pageObject';
-import faker from 'faker';
+  from '../support/pages/homeCatalogue.pageObject';
+import { faker } from '@faker-js/faker';
 /// <reference types='cypress' />
 
 const contactForm = new ContactFormPageObject();
 const homePage = new HomeAndCataloguePageObject();
 
 const testData = {
   email: faker.internet.email(),
-  name: faker.name.firstName(),
-  message: faker.random.words(),
+  name: faker.person.firstName(),
+  message: faker.word.words(),
```

Comment:
The 'faker.word.words()' method is used here. Ensure that this method is available in the version of the faker library you are using, as method names can vary between versions.